OSDOCS-6442 quick fix (#62744)
openshift/openshift-docs: modules/cli-logging-in-web.adoc

+
[source,terminal]
----

> $ oc login <cluster_url> --web --callback-port <port_number> <1>

Reviewer: ```suggestion
$ oc login <cluster_url> --web <1>
```

----

> <1> If needed, you can specify the server URL and callback port.

Reviewer: ```suggestion
<1> Optionally, you can specify the server URL and callback port. For example, `oc login <cluster_url> --web --callback-port 8280 localhost:8443`.
```

Author: Changes made, thanks!
